src/gui/entry/EditEntryWidget.h

```diff
@@ -62,7 +62,11 @@ class EditEntryWidget : public EditWidget
     explicit EditEntryWidget(QWidget* parent = nullptr);
     ~EditEntryWidget();
 
-    void loadEntry(Entry* entry, bool create, bool history, const QString& parentName, Database* database);
+    void loadEntry(Entry* entry,
+                   bool create,
+                   bool history,
+                   const QString& parentName,
+                   QSharedPointer<Database> database);
```

Comment:
I don't like stacked definitions, can you trim this up a little?

Reply:
Neither do I. I think I need t have a word with CLion.